qiskit/circuit/controlledgate.py

```diff
@@ -55,6 +56,8 @@ def __init__(
                 must equal num_ctrl_qubits, MSB on left. If None, use
                 2**num_ctrl_qubits-1.
             base_gate: Gate object to be controlled.
+            condition: tuple (ClassicalRegister, int), tuple (Clbit, bool) or
+                tuple (Clbit, int) for classically conditioned gate.
```

Comment:
`bool` subclasses `int`, so `(Clbit, bool)` is already compatible with a definition `(Clbit, int)`.

Reply:
For that reason maybe it's enough to just mark `int`.

Reply:
I copied this from elsewhere in the code but I'll simplify it.